### ISSUE TITLE
Rebalancing food

### ISSUE THREAD

**Issue by TheSyrethClan** [jeremyshannon/Exile#20]
Currently the food meta in Exile is mostly dependent onbaking maraka bread and cooking anperla, as these both give you in 24 hunger and 14 energy points a piece while allowing massive batches of them to be harvested. 

In comparison, cooked meat from small birds and small fish give 10/2, and Sakramos (which is a gigantic shark-like creatue) only gives 30/6. This makes hunting and animal husbandry vastly inferior and more time-consuming than harvesting or farming. The only 'farm' animal of any value currently is the Gundu as their eggs add 10 thirst and 40 hunger points and they reproduce at a fairly fast rate.

My suggestion is to reduce bread and anperla to 5/1 and add items that would be a combination of the above ingredients such as meat sandwhiches or anperla salads that would have 10/2 or 24/14 or higher values depending on complexity. This would ensure all mechanics of the game are used roughly evenly.

**Comment by jeremyshannon** [comment]
This has been on my mind, but it's kind of low priority. What I'd like to do is make the hunger system based on a very simplified caloric system and pin the various foods to rough real world analogues. Like maybe a goose for a pegasun (larger than a chicken, smaller than a turkey) or a crayfish for an impethu.
 That way I don't have to pull numbers entirely out of my butt, and things will sorta make some kind of sense.

 Higher priorities for me though are adding more kinds of creatures,  getting farming to work when the player's not present, fixing up multiplayer, etc. But if you want to gin up some numbers that make more sense than what we've got, go for it.

**Comment by DokimiCU** [comment]
Current values are roughly based on this:

Some notes:

Calculating sensible values for food:
(intervals/day) * hunger_rate = daily basal food needs
i.e. 20min/1min * 2 = 40 units per day

Therefore 40 units = 2000 calories.
calories -> units = 2000/40 = 50 cal/unit

Sugar 3900 cal/kg = 78 units/kg
Bread 2600 cal/kg = 52 units/kg
Potato. 1500 cal/kg = 30 u/kg
Meat 1000 cal/kg = 20 u/kg
cabbage 240 cal/kg = 4.8 u/kg

(notes from health file)

Because Exile only accounts for energy (and not protein, minerals vitamins, digestibility) it does bias carbohydrates. The optimal diet is pure sugar!

**Comment by wizardofgcc** [comment]
Foods could probably use some sort of approximate division between slow and fast carbohydrates, i. e. the maraka bread/anperla have their hunger expire quickly, while the meats may have their hunger last for a longer. Maybe could be done by altering hunger_rate for evertyhing temporarily. This could make players eat many kinds of food, combining meat and bread/anperla and other plants.

**Comment by TheSyrethClan** [comment]
I've been playtesting the version I modded to the best of my (admittedly limited) ability. So far what I've done was messing around with the values of certain items, adding some combined foods and adding a cutting board table type furniture to prepare them on. It definetly feels different, and I like how I need to hunt now to get more out of my food. Here are some numbers (thirst/hunger/energy):

- Tangkal - 5/10/10 - Unmodified. Served as a basis.
- Cooked Bread / Anperla - 1/10/2 - Nerfed. You can mass produce these to compensate.
- Mashed Anperla - 6/30/6 - 6x above
- Meat Strips - 1/10/2 - Strip cooked meats of bones and other undesirable parts for more carrying capacity. Spider gives x1, Pegasun and Gundu meat give x2, Sakramos give 4x.
- Cooked Pegasun and Gundu meat - 2/20/4 - Buffed to double value. Gundu still better because they spawn more with one egg block. Pegasun could compensate by laying them much more frequently.
- Cooked Sakramos - 4/60/12 - Buffed. It's a shark.
- Meat Sandwich - 2/36/14 - 2 pieces of bread + 1 meat strip. Big energy boost and you get some extra satiation from the taste than eating separately.
- Meat Sandwich w Zufani - 4/48/21 - 2 pieces of bread + 1 meat strip + 2 zufani. tufani is practically useless raw in its current state. This should make it more desirable as a topping.
- Meat Sandwich w Salad - 6/60/28 - 2 pieces of bread + 1 meat strip + 2 zufani + 1 galanta. Again, galanta is both useless and rare right now.
- Filled Baked Anperla - 2/36/14 - 1 Anperla + 2 meat strips. A viable alterative for sandwiches if no trees are nearby.

**Comment by wizardofgcc** [comment]
Another thing about food currently is that the hunger indicator is misleading. The percentage display makes it look like you need it to be as high as possible, which is not really the case. There is in fact a penalty for being too full (i. e. you speed is reduced). Maybe some sort of daily/short term nutrition display, or tweaking the colours so that high values will be yellow/orange would help emphasize that fact? Although in any form, the current percentage hunger value should probably stay, it gives a good long-term indicator of your food condition.

**Comment by jeremyshannon** [comment]
> There is in fact a penalty for being too full (i. e. you speed is reduced).

Is there? The comments in health/init.lua suggest it, but down in the code if hunger > 800 then it adds 1 to move/jump rate. There's some code duplication going on, too, as this bonus/penalty is also applied in on_actions.

**Comment by wizardofgcc** [comment]
Yes, that seems to be the case. I guess it was an intended feature initially, then got scrapped.
